src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AspNet/ApiController_ExecuteAsync_Integration.cs

```diff
@@ -3,21 +3,20 @@
 // This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
 // </copyright>
 
-#if NETFRAMEWORK
 using System;
 using System.Collections.Generic;
 using System.ComponentModel;
 using System.Linq;
 using System.Threading;
 using System.Web;
 using Datadog.Trace.ClrProfiler.CallTarget;
-using Datadog.Trace.ClrProfiler.Emit;
 using Datadog.Trace.ClrProfiler.Integrations;
+#if NETFRAMEWORK
 using Datadog.Trace.ClrProfiler.Integrations.AspNet;
+#endif
 using Datadog.Trace.Configuration;
 using Datadog.Trace.DuckTyping;
 using Datadog.Trace.ExtensionMethods;
-using Datadog.Trace.Headers;
 using Datadog.Trace.Tagging;
```

Comment:
I assume this change is related to generated the integrations.json/cs file? I don't think this is necessary, as we look through all of the dlls for the different platforms and do a `Distinct()`? (I prefer the #if def over the whole file, makes more sense I think)

Reply:
I think I can rollback this, this was done because before I was extracting the integrations using reflection, and depending on the Tracer dll loaded I was having different numbers of integrations.... Not a problem anymore.

Reply:
done in 7dcf857